lib/tasks/cards.rake

```diff
@@ -108,6 +108,96 @@ namespace :cards do
         is_unique: card["is_unique"],
         display_subtypes: flatten_subtypes(subtypes, card["subtypes"]),
       )
+
+      # Look for specific abilities and attributes:
+      if new_card.text 
+        m = new_card.text.match(/\+([X\d]+)\[link\]/)
+        if m && m.captures.length == 1
+          link_provided = m.captures[0]
+          puts "provides_link: [%s] [%s] %s\n-----------------\n" % [link_provided, new_card.title, new_card.text]
+          # Null is equivalent to "does not provide link" and we will use -1 to map to X.
```

Comment:
That's an interesting thought. Do you have an example?

Reply:
is it unexpected, though?  If i search with provides_mu! 2, X is definitely different from 2.

provides_mu<2 is different because we need to decide how to treat X for sure, but i think we can do that in the query builder for the search operator.

Reply:
By inequalities I meant > and <. Technically X is treated as 0 by cards that care about printed cost, so I'd probably implement the search operator that way.

Reply:
If you want an example, without manually overriding it searching `o<0` will get the [7 cards](https://netrunnerdb.com/find/?q=o%3Ax) that have a play/install cost of X.

Reply:
-1 vs. treating it as 0 is a no-op for you < and > examples.    foo<2 would catch the same cards whether we have the current -1 or treat it as 0.  i'd still rather have -1 as the translation for X.